### ISSUE TITLE
AttributeError: 'Record' object has no attribute '_file_indices'

### ISSUE THREAD

**Issue by tiborsimko** [cernopendata/cernopendata-portal#64]
This used to work well a couple of weeks ago:

```console
$ cd opendata.cern.ch
$ docker compose up
$ docker exec -i -t opendatacernch-web-1 /code/scripts/populate-instance.sh --skip-records --skip-glossary --skip-docs
$ docker exec -t opendatacernch-web-1 cernopendata fixtures records --mode insert-or-replace -f /content/data/records/delphi-documentation-manuals.json
$ firefox http://127.0.0.1:5000/record/80506
```

Currently, after pulling new container images:

```console
$ docker compose down -v
$ docker compose pull
```

the same process now leads to an internal server error when trying to serve files:

![e](https://github.com/user-attachments/assets/c2e0cc68-f0cf-4ead-bdc1-c52229a84e2d)

Indeed:

```console
web-1                    |     return view_method(pid, record, template=template, **kwargs)
web-1                    |   File "/code/cernopendata/modules/records/utils.py", line 166, in record_file_page
web-1                    |     index_files = record._file_indices
web-1                    | AttributeError: 'Record' object has no attribute '_file_indices'
web-1                    | 172.19.0.1 - - [07/Oct/2024 16:24:12] "GET /glossary/json HTTP/1.1" 200 -
```

This seems to be due to indexes not being created during site creation, as per the message:

```
...
There is no need for 'Creating indexes' (no items specified)
...
```

I think we should always create templates, indexes etc when creating the site, since these aren't added later when populating the site.

**Comment by tiborsimko** [comment]
BTW the same happens when one does:

```console
$ docker exec -i -t opendatacernch-web-1 /code/scripts/populate-instance.sh
...
There is no need for 'Putting templates' (no items specified)
Putting component templates...
...
Putting index templates...
...
There is no need for 'Creating indexes' (no items specified)
```

**Comment by psaiz** [comment]
Hi Tibor, 
Thanks for the ticket. The part of the indices is not related to this issue. The indices are created automatically when the data is inserted. The `search init` uploads the templates for the indices, so that once they are created, they have the proper mappings.

The root cause of the issue should be addressed by github.com/cernopendata/cernopendata-portal/pull/62. Until that is addressed, please  use the tag `v0.1.11`, which is the one deployed in production. 

Best, 
pablo

**Comment by tiborsimko** [comment]
Thanks, 0.1.11 works perfectly :+1: 

I'll pin the image in the `opendata.cern.ch` repository. We could revert back when the core issue is fixed, however for content curators it may be best to keep the image version pinned in the content repository and upgrade it only from time to time to well-defined stable versions, rather than living on the bleeding-edge...
